products/monitoring/terraform.yaml

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Provider::ResourceOverrides
+  AlertPolicy: !ruby/object:Provider::Terraform::ResourceOverride
+    exclude: false
+    id_format: "{{name}}"
+    examples: |
+      ### Basic Usage
+      ```hcl
+      resource "google_monitoring_alert_policy" "basic" {
+        display_name = "Test Policy Basic"
+        combiner = "OR"
+        conditions = [
+          {
+            display_name = "test condition"
+            condition_threshold {
+              filter = "metric.type=\"compute.googleapis.com/instance/disk/write_bytes_count\" AND resource.type=\"gce_instance\""
+              duration = "60s"
+              comparison = "COMPARISON_GT"
+              aggregations = [
+                {
+                  alignment_period = "60s"
+                  per_series_aligner = "ALIGN_RATE"
+                }
+              ]
+            }
+          }
+        ]
+      }
+      ```
+    properties:
+      name: !ruby/object:Provider::Terraform::PropertyOverride
+        custom_expand: 'templates/terraform/custom_expand/monitoring_alert_policy_name.erb'
+        custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      post_create: templates/terraform/post_create/alert_policy.erb
+
+  Group: !ruby/object:Provider::Terraform::ResourceOverride
+    exclude: true
```

Comment:
It's because it isn't ready. I added details [in the commit](https://github.com/GoogleCloudPlatform/magic-modules/pull/503/commits/9e47223eec4d85bde6fff74146c45d2ee830320a) but a comment makes a lot more sense in hindsight :P 

TBH the only reason I left this in here rather then stripping it or commenting it out was so that I could preserve the git blame history. However I don't feel super strongly about it. I'm happy to go with whichever you prefer (add comment+exclude or rip it out and add back later).

Reply:
Ok! I'm also fine either way. Let's do comment+exclude then (but put the exclude in api.yaml)